vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_value_util_unittest.cc

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_value_util.h"
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "base/test/values_test_util.h"
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace {
+constexpr char kJson[] =
+    R"({"locale": "EN", "timestamp": "2022-06-09 08:00:00.704847", "version": 1, "embeddings": {"quick": [0.7481, 0.0493, -0.5572], "brown": [-0.0647, 0.4511, -0.7326], "fox": [-0.9328, -0.2578, 0.0032]}})";
+}  // namespace
+
+namespace ads::ml::pipeline {
+
+class BatAdsPipelineEmbeddingValueUtilTest : public UnitTestBase {
```

Comment:
Some objects are known as `PipelineEmbedding` and others `EmbeddingPipeline`, is this intentional?